src/btree/bt_compact.c

```diff
@@ -110,6 +110,13 @@ __compact_progress(WT_SESSION_IMPL *session)
           bm->block->compact_pages_written);
         session->compact->prog_msg_count++;
     }
+
+    WT_STAT_CONN_INCRV(
+      session, session_table_compact_pages_reviewed, bm->block->compact_pages_reviewed);
+    WT_STAT_CONN_INCRV(
+      session, session_table_compact_pages_skipped, bm->block->compact_pages_skipped);
+    WT_STAT_CONN_INCRV(
+      session, session_table_compact_pages_written, bm->block->compact_pages_written);
```

Comment:
I'm not an expert on how stats work, I don't understand how the stats above is reset when we call compact next time? Do we need to do it manually at the beginning of `__wt_session_compact`?